content/Lab2/20_Testing/1_test_setup.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Test Setup"
+weight: 1
+---
+
+## Test Setup
+
+To test this setup we will deploy 2 linux machines. One will act as a bastion server to which the user will SSH to. The other machine will be the test machine (without direct internet connection) from which the test traffic to the internet will be generated.
+
+### **1. Deploy the Bastion** 
```

Comment:
is it possible to use terraform to deploy the bastion, since the rest of the lab is deployed with terraform?